.vscode/tasks.json

```diff
@@ -5,7 +5,7 @@
     "tasks": [
         {
             "label": "Build Test (Config: Debug-LowTrust)",
-            "command": "dotnet build Test/Test.fsproj -c Debug-LowTrust -p:Platform='Any CPU'",
+            "command": "dotnet build Test/Test.fsproj -c Debug-LowTrust -p:Platform=AnyCPU",
```

Comment:
Is there a reason you specify `AnyCPU`? It usually is the default.

Reply:
I'm not sure the switch is redundant and didn't want to spend time checking this :-)